### ISSUE TITLE
`no-misused-object-entries`

### ISSUE THREAD

**Issue by ljosberinn** [sindresorhus/eslint-plugin-unicorn#1171]
Using `Object.entries` to loop over objects may be convenient, but is sometimes overlooked in favor of more appropriate methods such as `Object.keys`/`Object.values` when `Object.entries` omits either key or value.

## Fail

```js
// prefer Object.keys
for(const [foo] of Object.entries(myObj)) {}

Object.entries(myObj).map(([foo]) => {})

// prefer Object.values
for(const [, foo] of Object.entries(myObj)) {}

Object.entries(myObj).map(([, foo]) => {})
```

## Pass

```js
// prefer Object.keys
for(const foo of Object.keys(myObj)) {}

Object.keys(myObj).map((foo) => {})

// prefer Object.values
for(const foo of Object.values(myObj)) {}

Object.values(myObj).map((foo) => {})
```

**Comment by fisker** [comment]
Duplicated of #1066?

**Comment by ljosberinn** [comment]
ha, I should've searched better, nice! same for #1172
